index.js

```diff
@@ -27,13 +32,19 @@ console.debug('About to generate "/tmp/generated-theme.less" tmp file:', content
 
 exec(`echo "${content}" > /tmp/generated-theme.less`, { cwd: process.cwd() }, (error1, stdout, stderr) => {
   if(!error1) {
-    console.debug(`Dynamic generation: "/tmp/generated-theme.less" tmp file was successfully generated`);
+    if(debug){
+      console.debug(`Dynamic generation: "/tmp/generated-theme.less" tmp file was successfully generated`);
+    }
     exec(
       `node_modules/less/bin/lessc --js /tmp/generated-theme.less ${scriptVariables.generatedThemeFilePath}`,
       { cwd: __dirname },
       (error2, stdout, stderr) => {
         if(!error2) {
-          console.log(`Finally: "${scriptVariables.generatedThemeFilePath}" was successfully generated`);
+          if(debug){
+            console.log(`Finally: "${scriptVariables.generatedThemeFilePath}" was successfully generated`);
+          }else{
+            console.debug(`AntDesign file (${process.argv[4]}) was successfully generated`);
```

Comment:
`process.argv[4]` should be: `scriptVariables.generatedThemeFilePath`. I mean, what if the user is using the defaults?